server/models/Topic.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
```

Comment:
same here

Reply:
For topic i changed it already, but for Blog i want to see if it works in that way because in mongoose's website i found a blogpost schema like below:

import mongoose from 'mongoose';
const { Schema } = mongoose;

const blogSchema = new Schema({
  title: String, // String is shorthand for {type: String}
  author: String,
  body: String,
  comments: [{ body: String, date: Date }],
  date: { type: Date, default: Date.now },
  hidden: Boolean,
  meta: {
    votes: Number,
    favs: Number
  }
});

ref: https://mongoosejs.com/docs/guide.html